serverless/functions/serverless-developing-go-functions.adoc

```diff
@@ -0,0 +1,27 @@
+include::modules/serverless-document-attributes.adoc[]
+[id="serverless-developing-go-functions"]
+= Developing Golang functions
+:context: serverless-developing-go-functions
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+:FeatureName: {FunctionsProductName}
+include::modules/technology-preview.adoc[leveloffset=+2]
+
+After you have xref:../../serverless/functions/serverless-functions-getting-started.adoc#serverless-functions-getting-started[created a Golang function project], you can modify the template files provided to add business logic to your function.
```

Comment:
This linked to the top of the "Getting started" assembly. Should it have linked directly to the module that discussed creating a project?

https://deploy-preview-32539--osdocs.netlify.app/openshift-enterprise/latest/serverless/functions/serverless-functions-getting-started.html#serverless-create-func-kn_serverless-functions-getting-started